src/injector/inject.ts

```diff
@@ -315,26 +361,58 @@ export function injectVM (vm: EurekaCompatibleVM) {
         vm.emit('LOCALE_CHANGED', locale);
         return result;
     };
-    // TODO: compiler support
-    const originalArgReporterBooleanFunc = vm.runtime._primitives.argument_reporter_boolean;
-    vm.runtime._primitives.argument_reporter_boolean = function (
-        args: Record<string, unknown>,
-        ...otherArgs: unknown[]
-    ) {
-        const eurekaFlag = args.VALUE;
-        switch (eurekaFlag) {
-            case '🧐 Chibi?':
+
+    const checkEureka = (eurekaFlag: string): boolean | null => {
+        switch (eurekaFlag.toLowerCase()) {
```

Comment:
seems there's no needed to make it case-insensitive, there's any reason?

Reply:
https://github.com/TurboWarp/scratch-vm/blob/99ff662f7fee8c3adebb7f1803c854773cf3218b/src/blocks/scratch3_procedures.js#L121

Reply:
seems it's just TurboWarp's alias